apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,389 @@
+                          data-title="Sync Errors"
+                          data-content='<pre style="background-color: black" class="text-white rounded p-3 mb-0">${question.sync_errors_ansified}</pre>'
+                        >
+                          <i class="fa fa-times text-danger" aria-hidden="true"></i>
+                        </button>
+                      `
+                    : question.sync_warnings
+                      ? html`
+                          <button
+                            class="btn btn-xs mr-1 js-sync-popover"
+                            data-toggle="popover"
+                            data-trigger="hover"
+                            data-container="body"
+                            data-html="true"
+                            data-title="Sync Warnings"
+                            data-content='<pre style="background-color: black" class="text-white rounded p-3 mb-0">${question.sync_warnings_ansified}</pre>'
+                          >
+                            <i
+                              class="fa fa-exclamation-triangle text-warning"
+                              aria-hidden="true"
+                            ></i>
+                          </button>
+                        `
+                      : ''}
+                  ${question.display_name}
+                </td>
+                <td>
+                  ${renderEjs(__filename, "<%- include('../partials/topic'); %>", {
+                    topic: question.topic,
+                  })}
+                </td>
+                <td>
+                  ${renderEjs(__filename, "<%- include('../partials/tags'); %>", {
+                    tags: question.tags,
+                  })}
+                </td>
+                <td>
+                  ${maxPoints({
+                    max_auto_points: question.max_auto_points,
+                    max_manual_points: question.max_manual_points,
+                    points_list: question.points_list,
+                    init_points: question.init_points,
+                  })}
+                </td>
+                <td>${question.max_manual_points || '—'}</td>
+                ${showAdvanceScorePercCol
+                  ? html`
+                      <td
+                        class="${question.assessment_question_advance_score_perc === 0
+                          ? 'text-muted'
+                          : ''}"
+                        data-testid="advance-score-perc"
+                      >
+                        ${question.assessment_question_advance_score_perc}%
+                      </td>
+                    `
+                  : ''}
+                <td>
+                  ${question.mean_question_score
+                    ? `${question.mean_question_score.toFixed(3)}`
+                    : ''}
+                </td>
+                <td>
+                  ${question.avg_question_score_perc
+                    ? `${question.avg_question_score_perc.toFixed(3)}`
```

Comment:
In the original template, this had a trailing `%`; should we add that back? Or maybe consider adding it above to `mean_question_score` as well?

Reply:
Added them in both places in [3eea193](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/3eea19357af59205e329d4566036dda743b2e285).